src/components/AccountDetails/Wallets.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+const defaultData = {
+  owner: {
+    phone: "+11111111111",
+  },
+  username: "username",
+  level: "ONE",
+  status: "ACTIVE",
+  title: "title",
+  coordinates: {
+    latitude: "13.4972747",
+    longitude: "-89.4435569",
+  },
+  createdAt: 1633992340,
+}
+
+function Wallets({ accountDetails, update, loading = false, updating = false }) {
+  const data = accountDetails || defaultData
```

Comment:
It seems that you are a following a pattern here with ```defaultData``` from other places in the repo, but I think it would be best if we move away from including testing data/logic in components.